extras/kms/repository.go

```diff
@@ -0,0 +1,242 @@
+package kms
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/hashicorp/go-dbw"
+	"github.com/hashicorp/go-kms-wrapping/extras/kms/v2/migrations"
+	wrapping "github.com/hashicorp/go-kms-wrapping/v2"
+	"github.com/hashicorp/go-kms-wrapping/v2/aead"
+	"github.com/hashicorp/go-secure-stdlib/strutil"
+	"github.com/hashicorp/go-uuid"
+)
+
+const (
+	// defaultLimit is the default for results
+	defaultLimit = 10000
+
+	// defaultWrapperSecret defines a default secret for testing
+	defaultWrapperSecret = "secret1234567890"
```

Comment:
If this is only for testing could you call it something like `testDefaultWrapperSecret`?

Reply:
done.  ty!